port/unix/omrsysinfo.c

```diff
@@ -1325,7 +1325,28 @@ omrsysinfo_get_s390_zos_supports_transactional_execution_facility(void)
 	uint8_t CVTFLAG4 = *(CVT + 0x17B);
 
 	/* Note we check for both constrained and non-constrained transaction support */
-	return OMR_ARE_ALL_BITS_SET(CVTFLAG4, 0x0C);
+	return OMR_ARE_ALL_BITS_SET(CVTFLAG4, 0x08);
+}
+
+/** @internal
+ *  Check if z/OS supports the Constrained Transactional Execution Facility (TXC). We use the CVTTXC (0x04) bit in
+ *  the CVT structure for the OS check.
+ *
+ *  @return TRUE if TX is supported; FALSE otherwise.
+ */
+static BOOLEAN
+omrsysinfo_get_s390_zos_supports_constrained_transactional_execution_facility(void)
+{
+	/* FLCCVT is an ADDRESS off the PSA structure
+	 * https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.iead300/PSA-map.htm */
+	uint8_t* CVT = (uint8_t*)(*(uint32_t*)0x10);
+
+	/* CVTFLAG4 is a BITSTRING off the CVT structure containing the CVTTX (0x08), CVTTXC (0x04), and CVTRI (0x02) bits
```

Comment:
I think these comments for this function and previous function to check non-constrained transactional facility can be commoned out.